tornjak-backend/api/agent/server.go

```diff
@@ -440,30 +459,37 @@ func (h spaHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 // HandleRequests connects api links with respective functions
 // Functions currently handle the api calls all as post-requests
 func (s *Server) HandleRequests() {
+	err := s.Configure()
+	if err != nil {
+		log.Fatal("Cannot Configure: ", err)
+	}
 	rtr := mux.NewRouter()
 
 	// Agents
-	rtr.HandleFunc("/api/agent/list", corsHandler(s.agentList))
-	rtr.HandleFunc("/api/agent/ban", corsHandler(s.agentBan))
-	rtr.HandleFunc("/api/agent/delete", corsHandler(s.agentDelete))
-	rtr.HandleFunc("/api/agent/createjointoken", corsHandler(s.agentCreateJoinToken))
+	rtr.HandleFunc("/api/agent/list", s.agentList)
+	rtr.HandleFunc("/api/agent/ban", s.agentBan)
+	rtr.HandleFunc("/api/agent/delete", s.agentDelete)
+	rtr.HandleFunc("/api/agent/createjointoken", s.agentCreateJoinToken)
```

Comment:
im assuming now that its serving from the same domain / server that CORS policy isnt required and that's why its removed yea? or was in put in a different way?